apps/next-evans/components/pages/explore/BookTeaserCards.tsx

```diff
@@ -72,7 +57,7 @@ const BookTeaserCards: React.FC<Props> = ({
             documentUrl={getDocumentUrl(book)}
             authorUrl={getFriendUrl(book.authorSlug, book.authorGender)}
             description={book.shortDescription}
-            badgeText={new Date(book.dateAdded).toLocaleDateString(`en-US`, {
+            badgeText={new Date(book.createdAt).toLocaleDateString(`en-US`, {
```

Comment:
i switched these `dateAdded`s back to `createdAt` for a more 1:1 correspondence with the database models. i found myself confused by the other names, and felt like consistency with the db was worth it here, since almost every one of our rows in the database have a `created_at` and `updated_at` timestamp